app/src/ui/lib/text-box.tsx

```diff
@@ -115,6 +152,14 @@ export class TextBox extends React.Component<ITextBoxProps, ITextBoxState> {
     }
   }
 
+  private onRef = (instance: HTMLInputElement) => {
```

Comment:
`instance` could be `null` as this also gets called when a component is unmounted.